### ISSUE TITLE
Install of GOG version of Unreal Tournament GOTY is not detected automatically

### ISSUE THREAD

**Issue by RoelerCoaster** [JosefNemec/Playnite#2001]
**Describe the bug**
I tried to automatically import my installed GOG games. Two of the games are imported fine, however one game is not detected. The game is [Unreal Tournament GOTY](https://www.gog.com/game/unreal_tournament_goty).

Some things I already cheked:
- I thought that it may be related to all the installed custom content, but I just did a fresh install of the game, but it is still not detected.
- I though it may somehow conflict with the game Unreal Tournament ("UT4")  from the epic games launcher, but also with Epic Games import disabled it is not detected. 

I can work around it by just adding the game manually, but I think it is a bit strange that this game specifically is not detected.

**Diagnostics ID**
8ecae874-c17b-41a8-a903-e6e5885743d2

**Comment by JosefNemec** [comment]
Can you try this? https://github.com/JosefNemec/Playnite/wiki/GOG-troubleshooting#some-games-appearing-as-uninstalled-in-playnite

**Comment by RoelerCoaster** [comment]
I don't use GOG galaxy as I always use the offline installers. But I just installed it and for whatever reason GOG doesn't detect the game either (the other games are detected). So I have now installed the game fresh through GOG Galaxy and it all seems fine, Playnite also detects it correctly now.

It seems that the offline installer for this particular game installs the game in a different way than GOG galaxy does. The other games work fine with just the offline installer. Anyway, thanks for pointing me in the right direction.
